toolset/benchmark/benchmarker.py

```diff
@@ -39,14 +39,16 @@ def run(self):
         # Get a list of all known  tests that we can run.
         all_tests = gather_remaining_tests(self.config, self.results)
 
+        all_passed = True
```

Comment:
Ugh, change this to something like `any_failed` and reverse the binary logic so that you can just bubble the return value up. That line of `return 0 if all_passed else 1` is so gross.